src/main/java/de/strifel/vbans/commands/CommandBan.java

```diff
@@ -26,13 +26,19 @@ public class CommandBan implements SimpleCommand {
     private final VBans vBans;
     private final String DEFAULT_REASON;
     private final String BANNED_BROADCAST;
+    private final String END_OF_HIS_LIFE;
 
     public CommandBan(VBans vBans) {
         this.server = vBans.getServer();
         this.database = vBans.getDatabaseConnection();
         this.vBans = vBans;
         DEFAULT_REASON = vBans.getMessages().getString("StandardBanMessage");
         BANNED_BROADCAST = vBans.getMessages().getString("BannedBroadcast");
+        if (vBans.getMessages().getString("EndOfHisLife")==null) {
+            END_OF_HIS_LIFE="";
+        } else {
+            END_OF_HIS_LIFE=vBans.getMessages().getString("EndOfHisLife");
+        }
```

Comment:
I think it might be better to use 
```suggestion
          END_OF_HIS_LIFE=vBans.getMessages().getString("EndOfHisLife", "the end of their life.");
```